src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -122,7 +122,10 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     executionResult match {
       case SuccessfulExecution(outputs, returnCode) => context.parent ! WorkflowActor.CallCompleted(key, outputs, returnCode)
       case AbortedExecution => context.parent ! WorkflowActor.AbortComplete(key)
-      case FailedExecution(e, returnCode) => context.parent ! WorkflowActor.CallFailed(key, returnCode, e.getMessage)
+      case FailedExecution(e, returnCode) =>
+        logger.error("Failing call: " + e.getMessage, e)
+        log.error(e, e.getMessage)
```

Comment:
I mean newcomer to the 'server' subcommand, not newcomer in general.  Many other foreground servers have console logging by default.  Having the server subcommand not print out anything and just sit there is less useful than a server that prints stuff out as its going.  I also like the consistency of saying "console logging is default" across the board and decoupling logging from the mode of operation.